docs/public/specifications/tests/Address-TP/address01.md

```diff
@@ -10,27 +21,47 @@ name servers must have addresses in the routable public addressing space.
 
 IANA is responsible for global coordination of the IP addressing system.
 Aside its address allocation activities, it maintains reserved address ranges
-for special uses. These ranges can be categorized into three types : 
-[Special purpose IPv4
-addresses](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xml),
-[Special purpose IPv6
+for special uses. These ranges can be categorized into three types:
+
+* [Special purpose IPv4
+addresses](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xml)
```

Comment:
To make the text easier to read, keep the link reference here `[Special purpose IPv4 addresses]` and put the following at the end of the document:

```
[Special purpose IPv4 addresses]:   https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xml
```

Applies of all cases with a URL.